CMP1903_A1_2324/Game.cs

```diff
@@ -15,7 +15,30 @@ internal class Game
          * rolls could be continous, and the totals and other statistics could be summarised for example.
          */
 
-        //Methods
+        /// <summary>  Creates three die objects, rolls them and outputs the values and their sum. </summary>
+        /// 
+        /// <returns> first die value, second die value, third die value, sum of all dice. </returns>
```

Comment:
Good use of XML commenting. Returns should be inside the function as it is not run upon instantiation.